kibana.yml

```diff
@@ -32,6 +32,8 @@
 # is proxied through the Kibana server.
 # elasticsearch.username: "user"
 # elasticsearch.password: "pass"
+%ELASTICSEARCH_USERNAME%
```

Comment:
Yes, in this case if env var is specified it'll basically add entire config line with it's value placed there. I can change the placeholder name if that helps?